webapp/src/domain/schemas/AllSchema.tsx

```diff
@@ -105,9 +128,19 @@ export default function AllSchema() {
                     <Table columns={columns}
                     data={list || []}
                     sortBy={[{ id: "name", desc: false }]}
-                     
+                    isLoading={loading}
                     />
+                     
                 </CardBody>
+                <CardFooter style={{ textAlign: "right" }}>
+                 <Pagination
+                        itemCount={schemas?.length || 0}
```

Comment:
I had not realised that `itemCount` was total count, so the only option for this to work correctly is for us to change the REST API

Reply:
like what?

Reply:
and we need to make change like- https://github.com/Hyperfoil/Horreum/blob/0c1147ff9d591dcfa655aeed09052e0694f1d502/horreum-model/src/main/java/io/hyperfoil/tools/horreum/api/ReportService.java#L74